e2e/test/scenarios/models/models-query-editor.cy.spec.js

```diff
@@ -45,7 +45,7 @@ describe("scenarios > models query editor", () => {
 
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
       cy.findByText("Row limit").click();
-      cy.findByPlaceholderText("Enter a limit").type("2");
+      cy.findByPlaceholderText("Enter a limit").type("2").blur();
```

Comment:
nit: even if calling `.blur` directly is handy, user will have to perform another action to trigger it, e.g. press tab or click outside, I'm not saying `.blur()` is incorrect, but maybe there is a better way to simulate blur by focusing another element following user behavior